src/Mod/BIM/bimcommands/BimMaterial.py

```diff
@@ -184,7 +184,7 @@ def Activated(self):
             context1.triggered.connect(self.onStartRename)
             context2 = self.contextMenu.addAction(translate("BIM", "Duplicate"))
             context2.triggered.connect(self.onDuplicate)
-            context3 = self.contextMenu.addAction(translate("BIM", "Merge to..."))
+            context3 = self.contextMenu.addAction(translate("BIM", "Merge to…"))
```

Comment:
```suggestion
            context3 = self.contextMenu.addAction(translate("BIM", "Merge To…"))
```

seems to be in a context menu. … is ok here!